README.md

```diff
@@ -55,26 +80,83 @@ cosign verify --key cosign.pub ghcr.io/avisi-cloud/structurizr-site-generatr:<ve
 
 Replace `<version>` with the version you want to verify.
 
-## Example usage
+## Usage
 
 These examples use the [example workspace](docs/example) in this repository.
 
-### Start a development server
+Once installed, Structurizr Site Generatr is operated via your terminal by issuing commands. Each command is explained here:
+
+### Help
+
+To learn about available commands, or parameters for individual commands, call Structurizr Site Generatr with the `--help` operator.
 
 ```shell
-structurizr-site-generatr serve --workspace-file docs/example/workspace.dsl
+installed> structurizr-site-generatr --help
+
+   docker> docker run -it --rm ghcr.io/avisi-cloud/structurizr-site-generatr:1.0.9 --help
```

Comment:
```suggestion
   docker> docker run -it --rm ghcr.io/avisi-cloud/structurizr-site-generatr:<version> --help
```

Reply:
Sure. This makes the document more cumbersome to read with the repetition, but I've reverted to the `<version>` + `Note:...` pairs.